subjects/rectperimeter/README.md

```diff
@@ -0,0 +1,43 @@
+## rectperimeter
+
+### Instructions
+
+Write a function that takes two `int`'s as arguments, representing the length of width and height of a rectangle and returning the perimeter of the rectangle.
+
+- If one of the arguments is negative it should return `-1`.
+
+### Expected function
+
+```go
+func RectPerimeter(w, h int) int {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"piscine"
+)
+
+func main() {
+	fmt.Println(piscine.RectPerimeter(10, 2))
+	fmt.Println(piscine.RectPerimeter(434343, 898989))
+}
```

Comment:
We should add one case for this condition: "If one of the arguments is negative it should return `-1`."

Reply:
fixed